cloud-config-server/server.go

```diff
@@ -46,8 +47,14 @@ func main() {
 		*caKey, *caCrt = certgen.GenerateRootCA("./")
 	}
 	// valid caKey and caCrt file is ready
-	candy.Must(fileExist(*caCrt))
```

Comment:
If there is no ca.pem or ca-key.pem the api-server.pem can not be generated correctly, so i don't think it's a right way that the cloud-config-server can be continued to run without ca.pem .